src/cloudsc_loki/CMakeLists.txt

```diff
@@ -416,9 +416,65 @@ if( HAVE_CLOUDSC_LOKI )
         OMP 1
     )
 
-    ##############################################################################
-    # C-transpilation mode for generating vectorized C host code (experimental!)
-    ##############################################################################
+    ####################################################
+    ##  SCC CUF (CUDA Fortran)                        ##
+    ##   * SCC with CUDA Fortran (CUF)                ##
```

Comment:
I think this should be wrapped in a `if( HAVE_CUDA )` block, otherwise loki-builds will fail desperately on anything but Nvidia hardware.